pkg/testutils/utils.go

```diff
@@ -160,14 +160,8 @@ func ModuleTemplateFactory(module v1alpha1.Module, data unstructured.Unstructure
 	return &moduleTemplate, nil
 }
 
-func readModuleTemplate(module v1alpha1.Module, moduleTemplate *v1alpha1.ModuleTemplate) error {
-	var template string
-	switch module.ControllerName {
-	case "manifest":
-		template = "operator_v1alpha1_moduletemplate_skr-module.yaml"
```

Comment:
To reviewer:
At the moment, we only deploy manifest in kcp cluster, right? I think we can remove this skr-module example to avoid confusion.

Reply:
manifests are deployed only in kcp but the target of the manifest can still be skr or kcp. For 2 cluster mode this is relevant I believe.

Reply:
clarified this change is correct.